src/routes/pricing/faq.svelte

```diff
@@ -8,36 +8,44 @@
-            answer: 'All current Cloud users will be notified about the pricing plans being active. Depending on your situation, you will either need to upgrade your account to a Pro plan, or you can happily continue on a Starter plan. You will receive an email once this occurs.'
+            question: 'Can I self-host Appwrite for free?',
+            answer: 'Yes! Appwrite Self Hosted will always be Open Source and free for you to use, without any limits. You can find the latest version of Appwrite on our GitHub repository.'
         },
         {
             question: 'Does Appwrite have a trial period?',
-            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can choose to either subscribe to the Pro plan or continue on a Starter plan.'
+            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can either subscribe to the Pro plan or continue on a Starter plan.'
         },
         {
             question: 'What payment methods does Appwrite support?',
-            answer: 'Appwrite currently supports credit and debit card payments. We will be working to constantly add support for more methods. Please reach out to us in case this is an issue for you.'
+            answer: 'Appwrite currently supports credit and debit card payments. We will be working to add support for more methods constantly. Please reach out to us in case this is an issue for you.'
+        },
+        {
+            question: 'What happens if I reach a resource limit in my Pro plan?',
+            answer: 'Your project will continue to run, and additional charges will apply. You can find the costs for additional resources under the pricing plans comparison. We will also notify you when you hit 75% and 100% of your resources with per email. You can also use our budget caps to avoid unexpected payments.'
         },
         {
-            question: 'What happens if I reach the storage limit in my Pro plan?',
-            answer: 'Your project will continue to run, and additional charges will apply. You can find the costs for additional storage under the pricing plans comparsions. We will also notify you when you hit 75% and 100% of your storage with an alert in the dashboard and per email. You can also use our budget tools to avoid unexpected payments.'
+            question: 'What happens if I reach a resource limit in my Starter plan?',
+            answer: 'Your project will freeze, and you will be requested to upgrade to Pro or downgrade the resources used in your project. We will also notify you when you hit 75% and 100% of your resources with per email.'
         },
         {
             question: 'How can I join the OSS program?',
             answer: 'The OSS program is exclusively for active open-source maintainers. In case you are liable for the program, you can <a href="/contact-us">apply here</a>.'
```

Comment:
Frontend issue but this link isn't easily visible as a link

![image](https://github.com/appwrite/website/assets/31401437/577694e4-50af-40ac-99c2-8f567f7ec067)

Reply:
cc: @TGlide